src/finance/finance.styled.js

```diff
@@ -0,0 +1,197 @@
+import styled from "styled-components";
+
+export const Financesection = styled.section``;
+
+export const Financecontainer = styled.div`
```

Comment:
you have to add one container in general file and use that for every section @Arbina001